fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraBinaryImpl.java

```diff
@@ -246,7 +248,7 @@ public URI getContentDigest() {
     public String getMimeType() {
         try {
             if (hasProperty(HAS_MIME_TYPE)) {
-                return getProperty(HAS_MIME_TYPE).getString();
+                return getProperty(HAS_MIME_TYPE).getString().replace(FIELD_DELIMITER + XSDstring.getURI(), "");
```

Comment:
Instead of hardcoding the clean-up of the `FIELD_DELIMITER` suffixes throughout the codebase, it seems cleaner to simply create a small utility in https://github.com/fcrepo4-exts/fcrepo4-upgrade-utils to be executed as part of the 4.6.0 to 4.7.0 upgrade process.